website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,99 @@
+## Color styles and tokens
+
+Color naming and use can vary depending on context and tooling. This overview will clarify what naming conventions are typically used and how they align to the HDS standards implemented in our libraries.
+
+In Figma, reusable colors are referred to as “Styles” and are categorized and stored in the “style panel” to be referenced in fills, borders, and other properties that accept color values. Styles directly reference a HEX value but are unable to reference each other in Figma (known as aliasing in token terminology). This means when we provide Figma styles for semantic usage (like `Foreground / Primary` with a value of `#3B3D45`), the HEX value is referenced directly instead of a base palette color (like `Palette / Neutral / Neutral 0`, which is also `#3B3D45`). Even though these two are the same, Figma’s current configuration requires manual maintenance if any of the base values are updated.
+
+In code we refer to “styles” as “Tokens” because they are generated from agnostic files (JSON in our case), and create atomic colors regardless of what platform or framework is used. For example, palette colors like `--token-color-palette-neutral-600` are directly referenced in `--token-color-foreground-primary`. This reduces the need to manually update all semantic values and their references to base palette colors through the automatic cascade of aliasing and inheritance.
+
+![Image showing a pyramid of how color inheritence works in tokens](/assets/foundations/color/colors-what-are-semantic-tokens.png)
```

Comment:
Ah, I see what you did there. This feels confusing because they are not 1:1 aliases, as the top two layers require some manipulation. Could you use an example where there is a direct 1:1 mapping for all levels? For example, you could do something like this: 

<img width="300" alt="color-token-abstraction" src="https://github.com/user-attachments/assets/e070387c-0f4b-4f55-a67b-5f18dd605102">

Reply:
I see you added the token and Figma mappings, but the colors used didn't change (two with opacity), which isn't an entirely accurate representation of how aliasing works. If needed, please work with @jorytindall tomorrow to find a more suitable example.

Reply:
Ok updated image to show what you have above